NTRightsNet.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.2</LangVersion>
+    <TargetFramework>net40</TargetFramework>
```

Comment:
Could you target .NET Standard 2.0 here or does the use of native calls make that nonsensical?

Reply:
For me .NET standard only makes sense for libraries, not executables.

Note that this project does _not_ use .NET Core in any way (see note below). It builds a .NET Framework executable and targets old-ish .NET 4.0 for the widest compatibility with clean OSes (i.e. no dependencies other than .NET Framework that comes pre-installed on all reasonably old Microsoft OSes).

I did try to build a self-contained .NET Core executable. ~60+ MB with zillion DLLs for "Hello world" program is just _stupid_ and unacceptable (contrast this with self-contained Go programs that are a single executable of a few megabytes).

_Note_: `dotnet build` in instructions might be confusing (it certainly was to me). It does not mean that the project is using .NET core per se. It is only using `dotnet` CLI tool (that happens to be part of .NET core distribution that Visual Studio 2017 installer happens to add to PATH). The `dotnet build` locates MSBuild 15.0 for us and triggers `Build` target on the default project file (`.sln` in our case).

In other words `dotnet build` is really just a fancier version of our current `tools/Run.cmd` or `plicli` MSBuild runners that should "just work" on all development machines with Visual Studio 2017.